WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductCoordinator.swift

```diff
@@ -146,6 +134,10 @@ private extension AddProductCoordinator {
     /// Returns `true` when there are existing products.
     ///
     func shouldShowGroupedProductType() -> Bool {
+        storeHasProducts()
+    }
+
+    func storeHasProducts() -> Bool {
```

Comment:
Nit: Can we keep `storeHasProducts` as a computed variable instead?

Reply:
Sure, updated in https://github.com/woocommerce/woocommerce-ios/pull/10208/commits/83f22f03a767c99a883c99aa8defd68260d7ee03